src/components/step/ConnectEhrStepContainer/ConnectEhrStepContainer.stories.tsx

```diff
@@ -1,20 +1,26 @@
 import React from "react";
-import { ComponentStory, ComponentMeta } from "@storybook/react";
-import ConnectEhrStepContainer, {
-    ConnectEhrStepContainerProps,
-} from "./ConnectEhrStepContainer";
+import { Meta, StoryObj } from "@storybook/react";
+import ConnectEhrStepContainer from "./ConnectEhrStepContainer";
+import { Card, Layout } from "../../presentational";
 
-export default {
+
+const meta: Meta<typeof ConnectEhrStepContainer> = {
     title: "SurveyStep/ConnectEhrStepContainer",
     component: ConnectEhrStepContainer,
     parameters: {
-        layout: "fullscreen",
-    },
-} as ComponentMeta<typeof ConnectEhrStepContainer>;
+        layout: 'fullscreen',
+        docs: {
+            //TODO: Description: <Description />
```

Comment:
Same question here regarding the TODO comment.  Can we remove it or implement the description?